Sources/Basic/OutputByteStream.swift

```diff
@@ -43,6 +43,9 @@ public protocol ByteStreamable {
 public class OutputByteStream: OutputStream {
     /// The data buffer.
     private var buffer: [UInt8]
+
+    /// Default buffer size of the data buffer.
+    private static let bufferSize = 1024
     
     public init() {
         self.buffer = []
```

Comment:
Huh, ok I guess I just assumed there was but see you are right. Can we use `repeating:count:` + `clear` instead so we can still kill the max below? This is inefficient, but we may want to move to manage our own memory anyway.